ironfish/src/wallet/wallet.ts

```diff
@@ -1519,9 +1521,24 @@ export class Wallet {
   }
 
   async importAccount(accountValue: AccountValue): Promise<Account> {
-    if (accountValue.name && this.getAccountByName(accountValue.name)) {
-      throw new DuplicateAccountNameError(accountValue.name)
+    let name = accountValue.name
+    if (accountValue.multisigKeys && isSignerMultisig(accountValue.multisigKeys)) {
+      const identityName = await this.getIdentityName(accountValue.multisigKeys.identity)
+      if (!identityName) {
+        throw new Error('Cannot import identity without an existing secret')
+      }
+
+      if (this.findAccountByIdentity(accountValue.multisigKeys.identity)) {
+        throw new Error('Account already exists with the provided identity')
```

Comment:
Is it really a problem to have two different accounts sharing the same multisig identity?